src/utils/user-agent-utils.ts

```diff
@@ -107,16 +107,18 @@ export const detectBrowser = function (user_agent: string, vendor: string | unde
         return MICROSOFT_EDGE
     } else if (includes(user_agent, 'FBIOS')) {
         return FACEBOOK + ' ' + MOBILE
-    } else if (includes(user_agent, CHROME)) {
-        return CHROME
-    } else if (includes(user_agent, 'CriOS')) {
-        return CHROME_IOS
     } else if (includes(user_agent, 'UCWEB') || includes(user_agent, 'UCBrowser')) {
         return 'UC Browser'
+    } else if (includes(user_agent, 'CriOS')) {
+        return CHROME_IOS // why not just Chrome?
```

Comment:
lost to the sands of time
we can check if there are any insights that reference this value and remove it 🤷

Reply:
I wonder if it's because chrome ios is just a wrapper around safari due to apple's engine restrictions, though in that case why not do the same for firefox? 🤷